src/components/forms/SearchBar/SearchBar.tsx

```diff
@@ -80,10 +81,10 @@ const SearchBar: React.FC<SearchBarProps> = ({
     const queryValue = searchInputRef.current.value;
     setSearchPerformed(false);
 
-    if (queryValue.length > 2) {
+    if (hasSuggestions && queryValue.length > 2) {
       setIsSearching(true);
-      const suggestions = await onSearch(queryValue);
-      setSearchResults(suggestions);
+      // const suggestions = await onSearch(queryValue);
```

Comment:
Commented code

Reply:
yes both the comments are to be fixed with suggestions

Reply:
Shouldn't be here performed the search? Otherwise it doesn't make much sense

Reply:
yes and to grab `suggestions` as a prop. Right now it's not functional, really just a placeholder

Reply:
adding TO DO comments to both :)